tests/test_datasets.py

```
@@ -163,6 +163,57 @@
+    with pytest.raises(Exception):
+        train_ds = SleapDataset(
+            slp_files=[two_flies[0]],
+            video_files=[two_flies[1]],
+            data_dirs="./data/sleap",
+            crop_size=[128, 128],
+            chunk=True,
+            clip_length=clip_length,
+            n_chunks=ds_length + 10000,
+        )
+
+    with pytest.raises(Exception):
```

Ruff (0.7.0)

`pytest.raises(Exception)` should be considered evil

(B017)